pandas/plotting/api.py

```diff
@@ -0,0 +1,20 @@
+"""
+Plotting api
+"""
+
+# flake8: noqa
+
+try:  # mpl optional
+    from pandas.plotting import converter
+    converter.register()  # needs to override so set_xlim works with str/number
+except ImportError:
+    pass
+
+from pandas.plotting.misc import (scatter_matrix, radviz,
+                                  andrews_curves, bootstrap_plot,
+                                  parallel_coordinates, lag_plot,
+                                  autocorrelation_plot)
+from pandas.plotting.core import (boxplot, scatter_plot, grouped_hist,
+                                  hist_frame, hist_series)
```

Comment:
The documentation argument is actually also true for `boxplot` and `scatter_plot`, and for both the functionality is also available in DataFrame.plot.scatter or DataFrame.boxplot/plot.box